addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -358,6 +356,19 @@ var FormRenderer = BasicRenderer.extend(WidgetAdapterMixin, {
             });
         }
     },
+    _applyFormSizeClass: function () {
+        const formEl = this.$el[0];
+        if (config.device.size_class <= config.device.SIZES.XS) {
+            formEl.classList.add('o_xxs_form_view');
+        } else {
+            formEl.classList.remove('o_xxs_form_view');
+        }
+        if (config.device.size_class >= config.device.SIZES.XXL) {
+            formEl.classList.add('o_xxl_form_view');
+        } else {
+            formEl.classList.remove('o_xxl_form_view');
+        }
+    },
```

Comment:
Changes in `/web` seems motivated by assertions in tests, but as I said they are irrelevant IMO, so I would revert this diff.

Reply:
Oh ok, didn't see enterprise branch before that. Intend was also to remove a media query, which is nice, but I feel that having more LOCs and mutating the DOM with `classList` is not a good alternative.

That's why I would still revert this and keep media query in enterprise. Less LOCs, and I would say such media-queries should be turned to OWL templates with `env` containing device screen size of some sort (out of scope of this PR OFC).

Reply:
except that if we keep the media query, none of this is testable. (except if these tests are moves to mobile_tests)

Again, in community there is no problem. But in enterprise, the media query will apply, even if the device size is forced, the chatter will still be on the side (without having the class) and none of the community behavior will be relevant in enterprise (the community tests are still executed in enterprise with enterprise code) . Except of course if I spawn a arbitrary attachment viewer for the sake of it....

Reply:
Hm... ok, all things considered this looks better than with just media query. Still has its share of problems, but the improvement on having tests with L/XXL size without attachment viewer is important considerable.